elements/lisk-p2p/src/peer.ts

```diff
@@ -55,6 +55,12 @@ interface ClientOptionsUpdated {
 	readonly connectTimeout?: number;
 }
 
+export type SCServerSocketUpdated = {
```

Comment:
If this is only used in `inbound_peer.ts`, it could be moved to that file and not exported.

Reply:
It is used also in the base class https://github.com/LiskHQ/lisk-sdk/blob/6c5ea4ca8da1d29addbcdbebab363d569b5ab7da/elements/lisk-p2p/src/peer.ts#L149 .